Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
It basically screws installing `npm` as we were doing it. As the author of the original language in this formula though, you can at least trust I understand what I'm trying to do 😄.